backend/accounts/models.py

```diff
@@ -9,6 +10,15 @@
 from phonenumber_field.modelfields import PhoneNumberField
 
 
+def get_user_file_name(instance, fname):
+    """
+    Returns a filepath unique to each username.
+    """
+    return os.path.join(
```

Comment:
Can we do input checking to make sure `fname` passed in is valid (nonempty, can be rsplitable, etc)